desktop-dde/dtk6gui/autobuild/defines

```diff
@@ -0,0 +1,12 @@
+PKGNAME=dtk6gui
+PKGSEC=dde
+PKGDEP="dtk6core gcc-runtime glibc librsvg qt-6"
+BUILDDEP="cmake ninja doxygen gtest"
+PKGDES="Graphical user interface development tools for deepin C++/Qt development"
```

Comment:
Tools or runtime libraries? Or both? If both, use "toolkit" or "toolkit and runtime".